solar_system.rb

```diff
@@ -0,0 +1,64 @@
+require_relative "planet"
+
+class SolarSystem
+  attr_reader :star_name, :planets
+
+  def initialize(star_name)
+    @star_name = star_name
+    @planets = []
+  end
+
+  def add_planet(planet)
+    @planets.push(planet)
+  end
+
+  def list_planets
+    planet_list = "Planets orbiting #{star_name}: "
+    i = 0
+    @planets.each do
+      planet_list += "\n#{i + 1}. #{@planets[i].name}"
+      i += 1
+    end
+    return planet_list
+  end
+
+  def find_planet_by_name(name)
+    i = 0
+    @planets.each do
+      if @planets[i].name.downcase == name.downcase
+        return @planets[i]
+      end
+      i += 1
+    end
+  end
+
+  def user_adds_planet
+    puts "Please complete the following information for the planet you would like added: \nName: "
+    name = gets.chomp.capitalize
+    puts "Color: "
```

Comment:
I like that you've broken this out as a separate method. Since this method interacts with the user, it might fit better in `main.rb`, possibly taking the solar system as an argument.